pkgs/tools/networking/clash-meta/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+buildGoModule rec {
+  pname = "clash-meta";
+  version = "1.14.2";
+
+  src = fetchFromGitHub {
+    owner = "MetaCubeX";
+    repo = "Clash.Meta";
+    rev = "v${version}";
+    sha256 = "sha256-sn+0TNXCK4af4zfkf09hLsFkuvkcyjhwh35kKo993FQ=";
+  };
+
+  vendorHash = "sha256-3j+5fF57eu7JJd3rnrWYwuWDivycUkUTTzptYaK3G/Q=";
+
+  # Do not build testing suit
+  excludedPackages = [ "./test" ];
+
+  CGO_ENABLED = 0;
```

Comment:
> Why disabling CGO.

Referring to https://github.com/NixOS/nixpkgs/blob/92247c104e4363e6dbb2ebb794cbfd7165b3031d/pkgs/tools/networking/clash/default.nix#L19
It could also successfully build with CGO enabled.

Reply:
I'll remove it and add tag to enable feature gvisor later.